bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="awattar" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>aWATTar Binding</name>
+	<description>aWATTar - Hourly Electricity Prices for Germany.</description>
```

Comment:
The binding name doesn't need to be repeated in the description.

Reply:
Changed